lms/djangoapps/completion/services.py

```diff
@@ -59,3 +61,10 @@ def get_completions(self, candidates):
             if candidate not in completions:
                 completions[candidate] = 0.0
         return completions
+
+    def get_completion_by_viewing_delay_ms(self):
+        """
+        Do not mark blocks complete-by-viewing until they have been visible for
+        the returned amount of time, in milliseconds.  Defaults to 0.
+        """
+        return getattr(settings, 'COMPLETION_BY_VIEWING_DELAY_MS', 0)
```

Comment:
Should we do `configuration_helpers.get_value('completion_by_viewing_delay_ms', getattr(settings, 'COMPLETION_BY_VIEWING_DELAY_MS', 0))` so this can be overridden per-site?

Reply:
I'm not familiar with configuration_helpers. What does it add over the pattern of allowing settings to be overridden from the lms.env.json file?

Reply:
It lets you change the setting via the django admin, and it lets you use different settings on different sites (i.e. white label sites).

But you know what, maybe if it ain't broke... This is probably fine as is.